packages/fxa-js-client/package.json

```diff
@@ -5,13 +5,13 @@
   "author": "Mozilla",
   "license": "MPL-2.0",
   "scripts": {
+    "postinstall": "grunt sjcl",
```

Comment:
nit: maybe add `publish: false`, because after this changed is added the release cannot be published anymore (it will fail to install via npm)